.travis.yml

```diff
@@ -11,5 +11,6 @@ script:
 - curl -L https://rawgit.com/InsightSoftwareConsortium/ITKPythonPackage/master/scripts/macpython-download-cache-and-build-module-wheels.sh -O
 - chmod u+x macpython-download-cache-and-build-module-wheels.sh
 - ./macpython-download-cache-and-build-module-wheels.sh
+- ./macpython-build-module-wheels.sh 2.7 3.5
```

Comment:
@jhlegarreta remove this line add add the version specification to the line above, i.e.

```
/macpython-download-cache-and-build-module-wheels.sh 2.7 3.5
```

Reply:
Done in 81eac07. Let's see if this makes the build timeout go away. Thanks Matt.